apps/local_ledger/test/local_ledger/entry_test.exs

```diff
@@ -138,6 +138,35 @@ defmodule LocalLedger.EntryTest do
       assert error.errors == [correlation_id: {"has already been taken", []}]
     end
 
+    test "returns a 'same address' error when the from/to addresses are identical" do
+      genesis()
+
+      res = Entry.insert(%{
+        "metadata" => %{},
+        "debits" => [%{
+          "address" => "mederic",
```

Comment:
Did you not try to avoid personal names? :D

Reply:
Oops, that's from a long time ago, will refactor in a future PR :D